capstone/static/js/search/temp_court_list.js

```diff
@@ -0,0 +1,2 @@
ne-cty-cir-ct", "Va.: Princess Anne Cty. Cir. Ct."], ["pulaski-cty-cir-ct", "Va.: Pulaski Cty. Cir. Ct."], ["radford-cir-ct", "Va.: Radford Cir. Ct."], ["radford-corp-ct", "Va.: Radford Corp. Ct."], ["rappahannock-cty-cir-ct", "Va.: Rappahannock Cty. Cir. Ct."], ["richmond-ch-ct", "Va.: Richmond Ch. Ct."], ["richmond-cir-ct", "Va.: Richmond Cir. Ct."], ["richmond-cty-cir-ct", "Va.: Richmond Cty. Cir. Ct."], ["richmond-hustings-ct", "Va.: Richmond Hustings Ct."], ["richmond-law-eq-ct", "Va.: Richmond Law & Eq. Ct."], ["roanoke-cir-ct", "Va.: Roanoke Cir. Ct."], ["roanoke-cty-cir-ct", "Va.: Roanoke Cty. Cir. Ct."], ["rockbridge-cty-cir-ct", "Va.: Rockbridge Cty. Cir. Ct."], ["rockingham-cty-cir-ct", "Va.: Rockingham Cty. Cir. Ct."], ["russell-cty-cir-ct", "Va.: Russell Cty. Cir. Ct."], ["salem-cir-ct", "Va.: Salem Cir. Ct."], ["scott-cty-cir-ct", "Va.: Scott Cty. Cir. Ct."], ["shenandoah-cty-cir-ct", "Va.: Shenandoah Cty. Cir. Ct."], ["smyth-cty-cir-ct", "Va.: Smyth Cty. Cir. Ct."], ["south-norfolk-corp-ct", "Va.: South Norfolk Corp. Ct."], ["southampton-cty-cir-ct", "Va.: Southampton Cty. Cir. Ct."], ["spotsylvania-cty-cir-ct", "Va.: Spotsylvania Cty. Cir. Ct."], ["stafford-cty-cir-ct", "Va.: Stafford Cty. Cir. Ct."], ["staunton-cir-ct", "Va.: Staunton Cir. Ct."], ["suffolk-cir-ct", "Va.: Suffolk Cir. Ct."], ["surry-cty-cir-ct", "Va.: Surry Cty. Cir. Ct."], ["sussex-cty-cir-ct", "Va.: Sussex Cty. Cir. Ct."], ["tazewell-cty-cir-ct", "Va.: Tazewell Cty. Cir. Ct."], ["va", "Va.: Va."], ["va-1", "Va.: Va."], ["va-cir-ct", "Va.: Va. Cir. Ct."], ["va-ct-app", "Va.: Va. Ct. App."], ["va-ct-spec-app", "Va.: Va. Ct. Spec. App."], ["va-gen-ct-1", "Va.: Va. Gen. Ct"], ["va-gen-ct", "Va.: Va. Gen. Ct."], ["va-gen-ct-2", "Va.: Va. Gen. Ct."], ["va-gen-dist-ct", "Va.: Va. Gen. Dist. Ct."], ["va-high-ct-ch", "Va.: Va. High Ct. Ch."], ["va-spec-ct-app-1", "Va.: Va. Spec. Ct. App."], ["va-super-ct-ch", "Va.: Va. Super. Ct. Ch."], ["virginia-beach-cir-ct", "Va.: Virginia Beach Cir. Ct."], ["warren-cty-cir-ct-2", "Va.: Warren Cty. Cir. Ct."], ["washington-cty-cir-ct", "Va.: Washington Cty. Cir. Ct."], ["waynesboro-cir-ct", "Va.: Waynesboro Cir. Ct."], ["westmoreland-cty-cir-ct", "Va.: Westmoreland Cty. Cir. Ct."], ["william-cty-cir-ct", "Va.: William Cty. Cir. Ct."], ["williamsburg-james-city-cty-cir-ct", "Va.: Williamsburg & James City Cty. Cir. Ct."], ["williamsburg-cir-ct", "Va.: Williamsburg Cir. Ct."], ["winchester-cir-ct", "Va.: Winchester Cir. Ct."], ["winchester-corp-ct", "Va.: Winchester Corp. Ct."], ["wise-cty-cir-ct", "Va.: Wise Cty. Cir. Ct."], ["wythe-cty-cir-ct", "Va.: Wythe Cty. Cir. Ct."], ["vt", "Vt.: Vt."], ["vt-dist-ct", "Vt.: Vt. Dist. Ct."], ["w-va", "W. Va.: W. Va."], ["w-va-ct-cl", "W. Va.: W. Va. Ct. Cl."], ["wash", "Wash.: Wash."], ["wash-ct-app", "Wash.: Wash. Ct. App."], ["wash-dist-ct", "Wash.: Wash. Dist. Ct."], ["wis-ct-app-1", "Wash.: Wis. Ct. App."], ["wis", "Wis.: Wis."], ["wis-ct-app", "Wis.: Wis. Ct. App."], ["wyo", "Wyo.: Wyo."]];
```

Comment:
Was this meant to be replaced by something else? Or is the goal to temporarily include a hardcoded courts list and replace it later?

Reply:
I was going to have this here, now, and replace it later with a snippet just to keep things moving. This was intended to be a quicker PR than it ended up being, though, so maybe I should just add the snippet functionality now?

Reply:
Yeah at a minimum with this much of a shortcut I would include a comment about what it's a placeholder for.

If adding a snippet is a lift to do now, I'd also note that the [Django cache framework](https://docs.djangoproject.com/en/3.2/topics/cache/) might be just as good or better for some of what we use snippets for (where rendering the real thing only takes a second or two anyway) -- either as a `{% cache None "court_list" %}` tag in the template, or using `cache.get` in the view.

Reply:
Good call— my bad. 

It looks like template fragment caching plus helper functions could be a drop-in replacement for snippets as we use them— I'm looking at the details RN. I need to look for specifics, but there might be one or two snippets for which that first load might be kind of a doozy. I think we could get around that by not automatically expiring caches and maybe having a fab task that runs immediately after deploy that requests pages with heavier data processing.

Reply:
Maybe we use cache for things that run in under a second like "dump 1000 entries for a dropdown list", and write files to /download/ for statistical reports that take longer to run. We didn't have the download section when we made the snippets thing, but now that we do it makes sense to write statistics like cases_by_decision_date to a csv in a statistics folder in downloads. Then the link can just be to that rather than to a database view. Let's chat about this offline as far as architecture ...